lib/parse.coffee

```diff
@@ -27,7 +27,7 @@ exports.parse = (argv) ->
 	result = {}
 
 	result.options = _.mapValues options, (value) ->
-		if /^[\d\.]+$/.test(value)
+		if /^\d+(\.\d+)?$/.test(value)
```

Comment:
Sent a PR for this - https://github.com/resin-io/capitano/pull/38 @Page- @jviotti